internal/cp/proxy.go

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright (c) 2008-2023, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package cp
+
+import (
+	"context"
+	"math"
+	"time"
+
+	"github.com/hazelcast/hazelcast-go-client/internal/cb"
+	"github.com/hazelcast/hazelcast-go-client/internal/cluster"
+	"github.com/hazelcast/hazelcast-go-client/internal/cp/types"
+	"github.com/hazelcast/hazelcast-go-client/internal/invocation"
+	"github.com/hazelcast/hazelcast-go-client/internal/logger"
+	"github.com/hazelcast/hazelcast-go-client/internal/proto"
+	"github.com/hazelcast/hazelcast-go-client/internal/proto/codec"
+	iserialization "github.com/hazelcast/hazelcast-go-client/internal/serialization"
+)
+
+/*
+proxy is the parent struct of CP Subsystem data structures.
+*/
+type proxy struct {
```

Comment:
Could you align the fields?

Reply:
Fixed at https://github.com/hazelcast/hazelcast-go-client/pull/901/commits/3604ce1f707a5e723a8c0241ba1108ccfad17b67.